BankingSystem.API/Extensions/ApplicationDependanciesConfiguration.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace BankingSystem.API.Extensions
+{
+    public static class ApplicationDependanciesConfiguration
+{
+    public static IServiceCollection ConfigureServices(this WebApplicationBuilder builder)
+    {
+        builder.Services.AddSwaggerGen();
+        builder.Services.AddAuthorization();
+        builder.Services.AddLogging(options =>
+        {
+            options.AddConsole();
```

Comment:
Please add Serilog library for our logging implementation

Reply:
Added